scripts/govcms-deploy

```diff
@@ -10,30 +10,34 @@ LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
 GOVCMS_DEPLOY_WORKFLOW_CONFIG=${GOVCMS_DEPLOY_WORKFLOW_CONFIG:-import}
 # Determine the content strategy `import` vs `retain`.
 GOVCMS_DEPLOY_WORKFLOW_CONTENT=${GOVCMS_DEPLOY_WORKFLOW_CONTENT:-retain}
-
-# Ensure tmp folder always exists.
-mkdir -p /app/web/sites/default/files/private/tmp/
+# The location of the application directory.
+APP="${APP:-$APP}"
 
 # Check for presence of config files.
 set +e # Prevent script failure when assigning 0.
-# shellcheck disable=SC2012
-config_count=$(ls -1 /app/config/default/*.yml 2>/dev/null | wc -l)
-# shellcheck disable=SC2012
-dev_config_count=$(ls -1 /app/config/dev/*.yml 2>/dev/null | wc -l)
+# shellcheck disable=SC2012,SC2086
+config_count=$(ls -1 $APP/config/default/*.yml 2>/dev/null | wc -l | tr -d ' ')
+# shellcheck disable=SC2012,SC2086
+dev_config_count=$(ls -1 $APP/config/dev/*.yml 2>/dev/null | wc -l | tr -d ' ')
 set -e
 
 echo "Running govcms-deploy"
 echo "Environment type: $LAGOON_ENVIRONMENT_TYPE"
 echo "Config strategy:  $GOVCMS_DEPLOY_WORKFLOW_CONFIG"
 echo "Content strategy: $GOVCMS_DEPLOY_WORKFLOW_CONTENT"
 echo "There are ${config_count} config yaml files, and ${dev_config_count} dev yaml files."
+
+# Ensure tmp folder always exists.
+mkdir -p "$APP/web/sites/default/files/private/tmp"
+
 drush core:status
 
 # Database updates, cache rebuild, optional config imports.
 common_deploy () {
 
   if [[ "$LAGOON_ENVIRONMENT_TYPE" = "development" && "$GOVCMS_DEPLOY_WORKFLOW_CONTENT" = "import" ]]; then
-    drush --alias-path=/etc/drush/sites sql-sync @ci.prod @self -y
+    echo "Performing content import."
```

Comment:
added a verbose message. also, used in test assertions. in general, every operation should notify the user that it happens

Reply:
Perhaps "Syncing database from production." or "Importing database from production."

Reply:
The messaging is consistent with line 48 below: "Performing config import."

Reply:
Sure but it's not really a content import, it's a database clobber.